persists/backend_server/Ballerina.toml

```
@@ -0,0 +1,17 @@
+[package]
+org = "bff"
+name = "persists"
+version = "0.1.0"
+distribution = "2201.8.2"
+
+[build-options]
+observabilityIncluded = true
+
+[persist]
+datastore = "mysql"
+module = "persists"
+
+[[platform.java11.dependency]]
```

```suggestion
[[platform.java17.dependency]]
```
Since we are using the update 8 shouldn't we use the java17 version?

---

updated